src/snowflake/snowpark/modin/plugin/compiler/snowflake_query_compiler.py

```diff
@@ -2017,33 +2017,32 @@ def _bool_reduce_helper(
 
         empty_value: bool
             The value returned for an empty dataframe.
-        reduce_op: {"and", "or"}
-            The name of the boolean operation to apply.
+        agg_func: {"all", "any"}
```

Comment:
will you be able to move this function to aggregation_utils to centralize all aggregation related helper and utils? you cam do this as a follow up pr